cmd/version.go

```diff
@@ -31,8 +31,15 @@ bbi version
 }
 
 func version(cmd *cobra.Command, args []string) {
-	fmt.Println(VERSION)
+	fmt.Println(buildVersionString(VERSION))
```

Comment:
So if you build it without LDFlags, it will not. The point is that when the releases are built with goreleaser, it's set at that point

```
builds:
  -
    id: "default"
    ldflags:
      - -s -w -X 'github.com/metrumresearchgroup/babylon/cmd.VERSION={{ .Env.VERSION }}'
```

The build operation is setting the VERSION variable in the CMD package to the content of the environment for Version in goreleaser's environment (which comes from Drone as the tag name)

The point of leaving it as develop is that it cannot be confused with release builds (which have identifiers) unless it's either done via goreleaser or LDFlags are set at build time. This is also how we're building Metrix currently.

Reply:
You can actually try it yourself in goland. If you set the LDFlags during build, and call version, it'll return whatever was specified during build